stdlib/hashtbl.mli

```diff
@@ -374,7 +374,7 @@ val hash_param : int -> int -> 'a -> int
    with the same properties as for [hash]. The two extra integer
    parameters [meaningful] and [total] give more precise control over
    hashing. Hashing performs a breadth-first, left-to-right traversal
-   of the structure [x], stopping after [meaningful] meaningful nodes
+   of the structure [x], stopping after [meaningful] nodes
```

Comment:
This was correct before - `[meaningful]` refers to the variable (a number) of meaningful nodes (a "meaningful" node is a definition). Imagine `[meaningful]` were `[n]`.